packages/bits/spec/components/progress/progress.e2e.ts

```diff
@@ -35,6 +37,22 @@ describe("USERCONTROL progress", () => {
             await buttonTooltip.waitToBeDisplayed();
             expect(await buttonTooltip.isTooltipDisplayed()).toBeTruthy();
             expect(await buttonTooltip.getTooltipText()).toContain("Cancel");
+            // Return to initial state
+            await indeterminateProgress.getCancelButton().click();
+            expect(await indeterminateProgress.canCancel()).toBe(false);
+        });
+    });
+
+    describe("compact progress", () => {
+        beforeEach(async () => {
+            await compactProgressBtn.click();
+        });
+
+        it("should show and hide progress bar when click button in compact progress", async () => {
+            expect(await compactProgress.isProgressBarDisplayed()).toBe(true);
+            expect(await compactProgress.hasClass("nui-progress--shown"));
```

Comment:
I'm wondering if these two queries are redundant? Do we need them both?

Reply:
You are right, Blake, I will delete one of them, that is the same.